src/custom-ci-info-bot.ts

```diff
@@ -0,0 +1,31 @@
+import * as probot from 'probot';
+
+// https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab
+const dummy = `
```

Comment:
I like that it's on the PR so people don't have to go hunting for it to find out what the labels are, but I think a) it should be a comment and not in the PR body and b) we should keep bot comments to a minimum (outside of the "*** has imported this PR" comments which do serve a purpose showing up multiple times).

So ideally this would be part of the Dr. CI comment, but I'm not sure how we'd implement that. We could have this all be internal like in [D29154762](https://www.internalfb.com/diff/D29154762), but that's meh. Probot also probably has permissions to edit the Dr. CI comment itself but I don't know if Dr. CI will play nicely with that.

Reply:
@driazati I asked @bigfootjon previously about putting it in the Dr. CI comment and he said that's not possible for technical reasons. I also remember @seemethere shares your concern about adding too many bot comments to the PR, which I think is also reasonable; so putting it on the PR description itself (in the Test Plan section at the bottom) seems the most reasonable option

Reply:
could people weigh in on this (e.g. @mruberry, @janeyx99, @zhouzhuojie)? I was under the impression that the consensus was to put it in the PR description itself, but I may have been misremembering

Reply:
I'd agree with this, I think since this is an opt in feature for people to use then it should be fine for it to be a separate comment

Reply:
just to clarify, the goal is for it eventually to _not_ be opt-in

Reply:
I'm agnostics to either description or a new comment for UX. However, technical implementation might be easier if it has its own comment block - which makes it simple to do a fresh update and can avoid race condition when people are editing the PR's description body. And agreed that Dr. CI's comment is the best place - but we need to dive deeper how to play nicely with that.

Reply:
Also, right now it's an opt-in experiment, so feel free to iterate on the ideas and get some feedback

Reply:
> I asked @bigfootjon previously about putting it in the Dr. CI comment and he said that's not possible for technical reasons.

Actually, thinking more about this, it's plausible that we could work around the technical limitations. If the consensus is to use Dr. CI comment bodies then I can help make that happen. Feel free to message me on workplace chat to work out the details.